Scripts/script-TaniumShowPendingActions.yml

```diff
@@ -148,3 +149,4 @@ dependson:
   must:
   - tn-get-object
 timeout: 0s
+releaseNotes: Deprecated script
```

Comment:
please write the command that replace this script.
Deprecated script, use `command name` instead.